setup.cfg

```diff
@@ -32,6 +32,8 @@ install_requires =
     psutil
     GPUtil
     importlib_metadata
+    psycopg2-binary
```

Comment:
We should be careful to not overload users with dependencies they may not use. I think it's fine to keep this in the requirements for now, for simplicity's sake. 

A longer term solution would be to just have a bare minimum set of dependencies always installed (e.g. numpy pandas torch, omegaconf) and have users install separate dependencies for the specific python tools (e.g. pyspark, psycopg2, mysql) or for the docs/testing (sphinx, breath, tox, pytest)

Reply:
I agree but I think a better way is to give users a simple option to select which tools they want to install and we install dependencies only for those tools. Rather than asking users to install.
For now, I have kept it as is.

Reply:
Marking as resolved as no action item